server/models/Applicant.js

```diff
@@ -0,0 +1,52 @@
+import pg from "pg"
+
+const pool = new pg.Pool({
+  connectionString: "postgres://postgres:password@localhost:5432/pet_adoption_agency"
+})
+
+class Applicant {
```

Comment:
I noticed the table that this relates to is `adoption_applications`, but the class is called `Applicant` which makes it sound like this is the person submitting an application.  I would rename the class to `AdoptionApplication` to better describe what this is representing.  Keep in mind you will also have to update anywhere else in your code you are calling on `Applicant`